tldw_Server_API/app/api/v1/endpoints/admin/admin_ops.py

```
@@ -646,6 +646,61 @@ async def delete_incident(
     return MessageResponse(message="incident_deleted")
 
 
+@router.post("/incidents/{incident_id}/notify")
+async def notify_incident(
+    incident_id: str,
+    request: Request,
+    principal: AuthPrincipal = Depends(get_auth_principal),
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)